WordPress/Classes/Utility/WPError.m

```diff
@@ -88,7 +89,7 @@ + (void)showNetworkingAlertWithError:(NSError *)error title:(NSString *)title
     }
 
     if (message == nil) {
-        message = [error localizedDescription];
+        message = [[error localizedDescription] stringByStrippingHTML];
```

Comment:
Gotcha. I was thinking we have a few places elsewhere in the app that call `showAlertWithTitle:message:withSupportButton:okPressedBlock:` and I'm honestly not sure if they pass an error description or something we've written.  Thanks for making the change. :)